Demo/Demo/Models/Order.swift

```diff
@@ -22,6 +23,13 @@ struct Order: Codable, Equatable {
         let attributes: Attributes?
     }
 
+    struct PayPal: Codable, Equatable {
```

Comment:
It doesn't look like we are using this struct anywhere currently, are there plans to use this in a future PR?

Reply:
I was thinking about our setupToken for vault without purchase for future. We have paymentSource struct and in iOS we have hard coded Card in there so I was thinking about this option.

Reply:
Without this struct,  compile error on response with PaymentSource because it is expecting this format on PayPal payments:
"payment_source": {
        "paypal": {
            "email_address": "sb-nxguf26143630@personal.example.com",
            "account_id": "2VVL8PDHGPNVJ",
            "account_status": "VERIFIED",
            "name": {
                "given_name": "John",
                "surname": "Doe"
            },
            "address": {
                "country_code": "US"
            }
        }
    }

Reply:
I see what you meant. I wasn't sure which fields I might use for displaying results for this PR or for the vault without purchase PR and I wanted to make it explicit that the PaymentSource we are using here is both for card payments and for PayPalPayment

Reply:
Maybe we can just keep `email_address` for now and display it somewhere similar to what we do for `lastDigits` with Card? We can always add to this when we get to the vault with purchase feature, but can wait to add to it until we decide what we actually need across iOS and Android for those views.

Reply:
Addressed here: e8d1cf5f0218bab1e18e3717dcf07275907e50a9